sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/AddressResolver.java

```diff
@@ -3,14 +3,14 @@
 
 package com.azure.cosmos.implementation.directconnectivity;
 
```

Comment:
Please consider sorting the imports here as one of them may now be out of order.

Reply:
Done